rotkehlchen/typing.py

```diff
@@ -317,3 +329,7 @@ class ExchangeApiCredentials(NamedTuple):
     api_key: ApiKey
     api_secret: ApiSecret
     passphrase: Optional[str] = None
+
+
+EXTERNAL_EXCHANGES: List = [Location.CRYPTOCOM, Location.BLOCKFI, Location.NEXO]
+EXTERNAL_LOCATION = [Location.EXTERNAL] + EXTERNAL_EXCHANGES
```

Comment:
nitpick: But let's change it in another PR. This is a not a good name. I am not even sure what a good description would be. But at least plural.

```suggestion
EXTERNAL_LOCATIONS_LIST = [Location.EXTERNAL] + EXTERNAL_EXCHANGES
```